packages/travelmakers-design-core/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,122 @@
+import { PolymorphicRef } from "@travelmakers-design-v2/styles";
+import { forwardRef, useState } from "react";
+import { View } from "../View";
+import useStyles from "./Input.style";
+import { InputComponent, InputProps } from "./Input.type";
+
+export interface Props extends React.HTMLAttributes<HTMLInputElement> {
+  label?: string;
+  subfix?: string | number;
+  feedback?: string;
+  isError?: boolean;
+}
+
+let defaultId = 0;
+
+export const Input: InputComponent & {
+  displayName?: string;
+} = forwardRef(
+  <C extends React.ElementType = "input">(
+    {
+      label,
+      subfix,
+      feedback,
+      isError = null,
+      value,
+      placeholder = "정보를 입력해주세요.",
+      autoComplete,
+      onClick,
+      onBlur,
+      onChange,
+      className,
+      ...props
+    }: InputProps<C>,
+    ref: PolymorphicRef<C>
+  ) => {
+    const [inputValue, setInputValue] = useState(value ?? "");
+    const [isFocus, setIsFocus] = useState(false);
+    const { classes, cx } = useStyles({ subfix, isError });
+    const [id] = useState(() => String(defaultId++));
+    const elementId = `tm-input-${id}`;
+
+    const onClickHandler = (e: React.MouseEvent<HTMLInputElement>) => {
+      setIsFocus(true);
+      onClick && onClick(e);
```

Comment:
p5; 이런 방식으로도 가능은 할 것 같아요~!
```suggestion
      onClick?.(e);
```

Reply:
간결하고 좋네요 ㅎㅎ 다 수정해버렸습니다!